### DISCUSSION TITLE
Converting a Private repository to Public repository does not create a Wiki section.

### DISCUSSION THREAD

**Discussion by lindorffs** [community/community#110443] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

As the title states.

With a Github Free account, converting an existing Private repository to a Public repository does not create the GitHub Wiki page which is created when a repository is created as Public from the get-go.

This was unexpected behavior to me.

**Comment by lindorffs** [comment]
This repository was created as private and then made public: https://github.com/lindorffs/private_to_public

This repository was created as public: https://github.com/lindorffs/public_to_public

This repository was created as public then converted to private, and made public once again: https://github.com/lindorffs/public_to_private_to_public

The third, public to private to public repository, does still have a Wiki page, despite once being private.

**Comment by lindorffs** [comment]
A quick and dirty solution is to [import](https://github.com/new/import) the repository after it is made public into a new public repository, delete the old repository, and import the new public version into a new public repository under the same name.

**Reply by lindorffs** [reply]
This is less than optimal however, upon documenting this, I found that the /assets/ directory does not transfer, and once the old repository is deleted, those files are gone.

**Reply by lindorffs** [reply]
https://github.com/lindorffs/private_public_fixed

**Reply by pot-belly-mole** [reply]
Thanks for this! Also, no need for the second import: what worked for me was import `myproject` repo to a public repo `myproject-temp`, then delete `myproject` repo and rename `myproject-temp` to `myproject`
